iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/HalfLineAdapter.java

```diff
@@ -32,14 +37,29 @@
     private double b2;
 
     public HalfLineAdapter(DanglingLine danglingLine, Branch.Side side, MergingViewIndex index) {
+        this(danglingLine, side, index, false);
+    }
+
+    public HalfLineAdapter(DanglingLine danglingLine, Branch.Side side, MergingViewIndex index, boolean mustBeReoriented) {
+
+        // TODO(mathbagu): is it correct? should we initialize only b1/g1 (resp. b2/g2) like it's done in the destructive merge?
+
+        // if mustBeReoriented is false then end1 and end2 of the halfLine correspond to end1 and end2 of the associated danglingLine
+        // if mustBeReoriented is true then end1 and end2 of the halfLine correspond to end2 and end1 of the associated danglingLine
+        // As in the danglingLine only G (g1 + g2) is recorded we do not need to know if the danglingLine has been reoriented
+        BranchReorientedParameters brp = new BranchReorientedParameters(danglingLine.getR(), danglingLine.getX(),
```

Comment:
I am not able to understand why we need this object here because the attributes are symmetrical. Why it is different that what `main` is doing?

Reply:
It is different that what you are doing in the `NetworkImpl`.

Reply:
You are right, it is the same. I decided to use the `BranchReorientedParameters` method to record that at this point if the shunt admittances are asymmetrical it is necessary to reorient the parameters. That can happen, in the future, if we extend the dangling line model by adding shunt admittances at both ends or if we decide not to split the shunt admittance.

In the NetworkImpl the shunt admittance of the danglingLine is not split so in this case the parameters must be reoriented. I do not know why the shunt admittance is not managed in the same way in both cases. These inconsistencies will disappear if the dangling line model is extended to support shunt admittance at both ends.

Reply:
Why are we making things different in the merging view implementation and in the in-memory implementation? Maybe we don't have to split B and G of the dangling line here, in the merging view and keep the reorientation.

Reply:
@miovd and @zamarrenolm too.
What I see is that we have two different behaviors depending on the implementation. In `NetworkImpl`, we have 
`BranchReorientedParameters brp2 = new BranchReorientedParameters(dl2.getR(), dl2.getX(), dl2.getG(), dl2.getB(), 0.0, 0.0);`
In the merging view, and more especially in the `HalfLineAdapter`, we have:
`BranchReorientedParameters brp = new BranchReorientedParameters(danglingLine.getR(), danglingLine.getX(),
 danglingLine.getG() / 2, danglingLine.getB() / 2, danglingLine.getG() / 2, danglingLine.getB() / 2, mustBeReoriented);`
We have two different behaviors, right? And the second one, in the merging view is a bi strange because in that case reorientation means nothing. I really prefer that we also fix that issue in that PR. Tie lines are a subject that is quite hard to maintain, I would like to make it as simple and clear as possible.

Reply:
Done. I fixed the second as the first one.